client/src/components/profileInfoEdit/ProfileInfoEdit.js

```diff
@@ -26,21 +26,10 @@ function ProfileInfoEdit () {
                             </Form.Text>
                         </Form.Group>
 
-                        <Row>
-                            <Col>
-                                <Form.Group controlId="formUserFirstName">
-                                    <Form.Label>First Name</Form.Label>
-                                    <Form.Control type="text" placeholder="John" />
-                                </Form.Group>
-                            </Col>
-                            
-                            <Col>
-                                <Form.Group controlId="formUserLastName">
-                                    <Form.Label>Last Name</Form.Label>
-                                    <Form.Control type="text" placeholder="Doe" />
-                                </Form.Group>
-                            </Col>
-                        </Row>
+                        <Form.Group controlId="formUserFirstName">
+                            <Form.Label>Research Group Name</Form.Label>
```

Comment:
`Thunder` is thinking about having two separate information, a Team name, as we have here and an Organisation Name. So for example, a team name maybe something the Research group chose, like DeepHackers, and the Organisation might be Monash or a corporate company. So could we add another field here to change the team's Organisation name?